hack/update-deps.sh

```diff
@@ -14,52 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-readonly ROOT_DIR=$(dirname $0)/..
-source ${ROOT_DIR}/scripts/test-infra/library.sh
-
 set -o errexit
 set -o nounset
 set -o pipefail
 
-cd ${ROOT_DIR}
-
-VERSION="release-0.18"
```

Comment:
not sure why client was still building for release 0.18.

Reply:
/assign

missed updating it back to master
currently the build tests are complaining, will fix in a separate PR

Reply:
PTAL #1082